go/saltpack/encrypt.go

```diff
@@ -17,9 +18,9 @@ type encryptStream struct {
 	header     *EncryptionHeader
 	payloadKey SymmetricKey
 	buffer     bytes.Buffer
-	nonce      *Nonce
 	inblock    []byte
-	tagKeys    []BoxPrecomputedSharedKey
```

Comment:
As in decrypt, we no longer need to save the precomputed shared secrets. Each pair of keys (receiver/ephemeral and receiver/sender) is only used once. (That's not *quite* true if the sender is anonymous. Then receiver/ephemeral is used twice. I didn't bother to avoid the second computation in that case, but we can if you want.